python/CMakeLists.txt

```diff
@@ -56,6 +56,9 @@ if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
   option(PYARROW_BUILD_JEMALLOC
     "Build the PyArrow jemalloc integration"
     OFF)
+  option(PYARROW_BUNDLE_ARROW_CPP
+    "Bundle the Arrow C++ libraries" 
+    OFF)
```

Comment:
Hmm, I'm hoping to soon move the `python/src/pyarrow` tree to `cpp/src/arrow/python`, any idea how this will be impacted or whether that will make things any easier?